kube-scheduler-healthcheck

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+set -eu
+TEST_POD_NAMESPACE="${TEST_POD_NAMESPACE:-kube-system}"
+TEST_POD_IMAGE="${TEST_POD_IMAGE:-gcr.io/google_containers/pause:3.0.0}"
+
+cat > /tmp/pod.yaml <<endOfPodDef
+apiVersion: v1
+kind: Pod
+metadata:
+  name: scheduler-pod-healthcheck
+spec:
+  containers:
+  - name: pause
+    image: ${TEST_POD_IMAGE}
+endOfPodDef
+
+alias delete_pod_command='kubectl delete pod scheduler-pod-healthcheck -n $TEST_POD_NAMESPACE --request-timeout=30s'
```

Comment:
sorry, it seems I was unclear.

The alias was for all instances of kubectl as they all use those same 2 flags, not just the delete pod one

`alias kubectl='kubectl --namespace $TEST_POD_NAMESPACE --request-timeout=30s'`

Reply:
aaaah ok, sorry I skim read the command in your previous comment